examples/data-frames.dx

```diff
@@ -123,158 +128,104 @@ def addRows {n m a} (t:n=>a) (t':m=>a) : (n|m)=>a =
     Left ni  -> t.ni
     Right mi -> t'.mi
 
--- > addRows(
---     students,
---     [
---       [row:
---         ("name", "Colton"), ("age", 19),
---         ("favorite color", "blue")]
---     ])
--- | name     | age | favorite color |
--- | -------- | --- | -------------- |
--- | "Bob"    | 12  | "blue"         |
--- | "Alice"  | 17  | "green"        |
--- | "Eve"    | 13  | "red"          |
--- | "Colton" | 19  | "blue"         |
--- > addRows(gradebook, [])
--- | name    | age | quiz1 | quiz2 | midterm | quiz3 | quiz4 | final |
--- | ------- | --- | ----- | ----- | ------- | ----- | ----- | ----- |
--- | "Bob"   | 12  | 8     | 9     | 77      | 7     | 9     | 87    |
--- | "Alice" | 17  | 6     | 8     | 88      | 8     | 7     | 85    |
--- | "Eve"   | 13  | 7     | 9     | 84      | 8     | 8     | 77    |
+:p addRows students [{name="Colton", age=19, favColor="blue"}]
```

Comment:
nit: I don't think you have to use `:p` anymore.

Also, once this is done, can you please check in the outputs, so that we can turn it into a test?

Reply:
Heh, I was only using it because I thought there was a reason you guys were using it.

Reply:
No, I think we should really delete it... I don't see a good reason to keep it.